### ISSUE TITLE
chat not work

### ISSUE THREAD

**Issue by zeus6999** [Hostname47/PHP_ENCRYPTED_CHAT#5]
What interests me chat that doesn't work.
If anyone knows how to make this chat work **I can pay for it**
if anyone can finish this script with chat Long polling he can contact me on Skype: crisdany7

**Comment by Hostname47** [comment]
You don't have to pay for it. Just put your error or what exactly is not working so maybe I can help you !

**Comment by zeus6999** [comment]
Hi, nothing works for me here. https://tinyurl.com/25wndfrj
I made two accounts
I can't make friends with each other.
I can't write any messages.

I don't know how to run and test CHAT

Regards

**Comment by Hostname47** [comment]
Hi,
Did you add the accounts as friends, you have to search for the second account, send a friend request and accept it in the first account, in order to be able to chat. Do that and see if it works !

**Comment by zeus6999** [comment]
Hi,
1) And what I did but no buttons work, and how it was not clickable
https://tinyurl.com/2727hjzw
Please download this script again on your computer and see which buttons do not work.

A thousand thanks, I'm waiting for news

2) I wanted to ask you, does Long polling chat work well?

Thanks,

**Comment by Hostname47** [comment]
Hello,
As far as I can remember, you can only chat with a friend if you add him as friend. Following only allow user to see his posts. I posted this project here just for learning purposes and it is not for production. I don't recommend long polling for chat apps, unless you have some simple interface between you and your users, but between users with each others, I recommend start using websockets.

if add button does not work, please check database table for requests and do some echos to pinpoint the error, but I'm sure it was working in the early days.

**Comment by Erneste1992** [comment]
thanks a lot. when open using mobile phone the web application not view well. means not see all application. can help mr

**Comment by Hostname47** [comment]
Yes the app is web based and not responsive to work well in mobile devices. However, PRs are welcomed if someone want to contribute and work on the mobile side or add anything that can make the app and look better

**Comment by Erneste1992** [comment]
thanks so much, how can send picture or file to the user in messaging box. please help methanks so much, how can send picture or file to the user in messaging box. please help me

**Comment by Erneste1992** [comment]
thanks so much, how can send picture or file to the user in messaging box.
please help me

**Comment by Erneste1992** [comment]
and how create a group chat
